### PR TITLE
Add KeyValue Service

### PR DESCRIPTION
**Pull request by polRk**

### CODE REVIEW

File: ydb_keyvalue_v1.proto [hunk 1]
```diff
@@ -2,10 +2,12 @@ syntax = "proto3";
 
 package Ydb.KeyValue.V1;
 
-option java_package = "tech.ydb.proto.draft.keyvalue";
-option go_package = "github.com/ydb-platform/ydb-go-genproto/draft/Ydb_KeyValue_V1";
+option go_package = "github.com/ydb-platform/ydb-go-genproto/Ydb_KeyValue_V1";
+option java_package = "tech.ydb.proto.keyvalue.v1";
+option java_outer_classname = "KeyValueGrpc";
```
Comment (alex268) [inline ydb_keyvalue_v1.proto:7]: java_outer_classname and java_multiple_files seem as unnecessary

File: ydb_table_v1.proto [hunk 2]
```diff
@@ -76,6 +76,7 @@ service TableService {
     // Streaming read table
     rpc StreamReadTable(Table.ReadTableRequest) returns (stream Table.ReadTableResponse);
 
+    // draft
```
Comment (alex268) [inline ydb_table_v1.proto:79]: Why? As I know this method is availabled for one year already

File: protos/ydb_common.proto [hunk 3]
```diff
@@ -4,8 +4,7 @@ option cc_enable_arenas = true;
 package Ydb;
 
 option go_package = "github.com/ydb-platform/ydb-go-genproto/protos/Ydb";
-option java_package = "tech.ydb.proto.common";
-option java_outer_classname = "CommonProtos";
+option java_package = "tech.ydb.proto";
```
Comment (asmyasnikov) [inline protos/ydb_common.proto:7]: Это вообще не стоило менять

File: protos/ydb_coordination.proto [hunk 4]
```diff
@@ -5,8 +5,6 @@ package Ydb.Coordination;
 
 option go_package = "github.com/ydb-platform/ydb-go-genproto/protos/Ydb_Coordination";
 option java_package = "tech.ydb.proto.coordination";
```
Comment (asmyasnikov) [inline protos/ydb_coordination.proto:7]: Аналогично

**Pull request closed without merging**